README.md

```diff
@@ -78,7 +84,26 @@ development: {
 
 #### Next, you will want to set up your database and start your server.
 
-Running `knex migrate:latest ` in the `server/` directory will use the migration files in `server/migrations` to create and format the tables so that they will work with wikonnect.
+Ensure that knex is installed globally to run some of the commands below
+Using npm:
+
+```
+npm install -g knex
+```
+
+Also ensure that you have redis-server running locally in your pc, To install it in your pc run the command below
+
+```
+sudo apt-get install redis-server
+```
+
+To confirm that the service is active, enter the command below
+
+```
+sudo service redis-server status
```

Comment:
Can add a statement for to for starting the server if it's not running:
```
sudo service redis-server start
```

Reply:
sure thing